UI Test Runner/UITests/FormViewTests.swift

```diff
@@ -1129,10 +1156,11 @@ final class FormViewTests: XCTestCase {
         let fieldTitle = app.staticTexts["switch integer"]
         let formTitle = app.staticTexts["mainobservation_ExportFeatures"]
         let formViewTestsButton = app.buttons["FormView Tests"]
+        let switchLabel = app.staticTexts["switch integer Switch Label"]
```

Comment:
Instead of using the switch's built in label (which would show right next to it) we place our on label next to it to get the horizontal seperation.

<img width="568" alt="Screenshot 2023-11-08 at 17 02 02" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/ddeb588c-9f36-414a-ba0a-7cf888ca6bb6">